specifyweb/frontend/js_src/lib/components/WorkBench/hooks.ts

```diff
@@ -568,6 +568,21 @@ export function getHotHooks(wbView: WbView) {
           10
         );
     },
+
+    /*
+     * Unallow user to select several times the same cell
```

Comment:
`Disallow user to select`/`Forbit user from selecting` would be better than `Unallow` in this context